utility/module.go

```diff
@@ -44,12 +60,75 @@ func (*utilityModule) Create(bus modules.Bus, options ...modules.ModuleOption) (
 
 	m.config = utilityCfg
 	m.mempool = types.NewTxFIFOMempool(utilityCfg.MaxMempoolTransactionBytes, utilityCfg.MaxMempoolTransactions)
+	m.logger = logger.Global.CreateLoggerForModule(m.GetModuleName())
+
+	return m, enableActorModules(cfg, m, bus)
+}
+
+// enableActorModules enables the actor-specific modules and adds them to the utility module's actorModules to be started later.
+func enableActorModules(cfg *configs.Config, m *utilityModule, bus modules.Bus) error {
```

Comment:
If you make this a receiver function of the module, there would be no need to bus in any parameters

```go
func (u *utilityModule) enableActorModules() error {
```